Updated docker-compose and Dockerfile

ExpressHermes opened the pull request:
**What have you Changed**

- Updated Dockerfile and docker-compose.yaml to run the app properly
- Added error handling in case the required env variables are not provided


### Issue no – #10 

### Self Check (Tick After Making pull Request)

- [x] One change in one Pull Request
- [x] I am following clean code and Documentation and my code is well linted with flake8.

Join Us on Discord:- https://discord.gg/JfbK3bS


## acknowledgement

- I have read and understood the above instructions. Your pull request will be counted as an open-source pull request and will be distributed under the MIT license.

ExpressHermes committed:
Updated docker-compose and Dockerfile

- Updated Dockerfile and docker-compose.yaml to run the app properly
- Added error handling in case the required env variables are not provided

ExpressHermes committed:
format file using autopep8 to fix linting issue

codeperfectplus commented on Dockerfile:
```diff
@@ -2,9 +2,9 @@ FROM python:3.9
 
-RUN start.sh
\ No newline at end of file
+CMD bahs start_server.sh
```
Spelling error for the bash. 

```
RUN bash start_server.sh
```
```
CMD bahs start_server.sh
```

I am not sure about cmd, cmd is for windows only commands right? will it work on Docker?

ExpressHermes replied:
Done

codeperfectplus commented on README.md:
````diff
@@ -65,14 +65,27 @@ Invite **Cybel** into your server
+2. Run the docker-compose command to run the bot
+
+```bash
+docker compose up
````
docker compose up -d
 
 to run in reattached mode

ExpressHermes replied:
Done

ExpressHermes committed:
Address review comments

- Fixed typo and changed `CMD` to `RUN` in Dockerfile
- Updated README.md file to run docker compose in detached mode
